src/openrct2/windows/ride.c

```diff
@@ -2138,7 +2138,8 @@ static void window_ride_show_ride_type_dropdown(rct_window *w, rct_widget *widge
     assert(_rideType == clamp(0, _rideType, RIDE_TYPE_COUNT));
 
     for (sint32 i = 0; i < RIDE_TYPE_COUNT; i++) {
-        gDropdownItemsFormat[i] = RideNaming[i].name;
+        gDropdownItemsFormat[i] = STR_DROPDOWN_MENU_LABEL;
+        gDropdownItemsArgs[i] = RideNaming[i].name;
```

Comment:
I just tested the code previously before "switch to dropdown_set_checked" and it worked, and there wasn't much changes since then so as long as it compiles it should work.